cmd/ocm/create/machinepool/cmd.go

```diff
@@ -40,6 +43,9 @@ var Cmd = &cobra.Command{
 	Long:    "Add a machine pool to the cluster.",
 	Example: `  # Add a machine pool mp-1 with 3 replicas and m5.xlarge instance type to a cluster
   ocm create machinepool --cluster mycluster --instance-type m5.xlarge --replicas 3 mp-1
+  # Add a machine pool mp-1 with autoscaling enabled and 3 to 6 replicas of m5.xlarge to a cluster
+  ocm create machinepool --cluster=mycluster --name=mp-1 --enable-autoscaling 
```

Comment:
You should end this line with `\`, otherwise a user might copy-paste and end up running two separate commands

Reply:
Fixed